shaka/src/js/mse/video_element.h

```diff
@@ -86,7 +86,9 @@ class HTMLVideoElement : public dom::Element {
   void SetCurrentTime(double time);
   double Duration() const;
   double PlaybackRate() const;
-  void SetPlaybackRate(double rate);
+  ExceptionOr<void> SetPlaybackRate(double rate);
```

Comment:
As I noted in another PR and you probably noticed, this won't work with the public `Video` type.  I think you could create a wrapper that returns `void` that just ignores the error for the `Video` version.